### DISCUSSION TITLE
set margin-header to null or false?

### DISCUSSION THREAD

**Discussion by davidleitch1** [quarto-dev/quarto-cli#9289] [Q&A] [answered]
### Description

Documentation shows how to add a subscription widget via margin-header in the root of the website. This results in it showing up on every page. I would like it to show up once, just on a "posts"  listing page. I have tried setting the individual posts to
``` 
margin-header: false
```
or 
``` 
margin-header: ""
```
or created a blank html "nothing.html" and set  on a post page eg "china.md"
```
margin-header: nothing.html
```
but they all either give compile errors (false and "") or have no effect.
I so like the cleanliness of quarto and the ease of building a website to my taste.

**Comment by mcanouil** [comment, marked as answer]
The option is a website level option.
I don't believe this to be available at a page level.

**Reply by davidleitch1** [reply]
Thank you. I guess the perfect is indeed the enemy of the good.

**Reply by mcanouil** [reply]
Note that you could tweak the code using JavaScript to display under certain conditions. (Out of scope of Quarto)
